<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="mainwidget" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="R3Label">
         <property name="text">
          <string>R3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="R1Label">
         <property name="text">
          <string>R1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="R2Label">
         <property name="text">
          <string>R2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="R2LineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="R0Label">
         <property name="text">
          <string>R0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="R0LineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="R1LineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="R3LineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="TimeLabel">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TimeLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="XLabel">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="YLabel">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ZLabel">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="XLineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ZLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="YLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="YawLabel">
         <property name="text">
          <string>Yaw</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="PitchLabel">
         <property name="text">
          <string>Pitch</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RollLabel">
         <property name="text">
          <string>Roll</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="YawLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="PitchLineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="RollLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
